lib/jmap/mail/mailbox/get/get_mailbox_method.dart

```diff
@@ -13,7 +13,7 @@ part 'get_mailbox_method.g.dart';
 @IdConverter()
 @AccountIdConverter()
 @PropertiesConverter()
-@JsonSerializable()
+@JsonSerializable(explicitToJson: true)
```

Comment:
why need `explicitToJson` ?

Reply:
@dab246 When I checked the method Mailbox/get, ReferenceIds in the request was malformed. You can check log below. So I added explicitToJson to solve this.
<img width="1327" alt="Screen Shot 2022-06-14 at 13 06 26" src="https://user-images.githubusercontent.com/107173849/173521856-6584d415-247c-4212-8391-5fdcbe4933bc.png">